src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -118,6 +142,14 @@ private TestProperties() {
         // access static fields directly
     }
 
+    public static boolean isCiEnvironment() {
+        return System.getenv("TRAVIS") != null || System.getenv("APPVEYOR") != null;
+    }
+
+    public static boolean isGodModeEnabled() {
+        return Boolean.parseBoolean(System.getProperty("godmode", "false"));
```

Comment:
This is not the best implementation since `TestProperties` values are final once initialized, but system properties are not. Consider the case when one activates GodMode with:
```java
System.setProperty("godmode", "true");
... // HTML verification
System.clearProperty("godmode");
```
This will create the same danger as when the one of test account values is `a`, albeit much less likely.

The only way to circumvent this is to remove the usage of system property altogether and moving it to `test.properties`, which is what #7379 is about. Noting that, this can proceed, but be sure to remedy this as soon as possible.